WolvenKit.App/ViewModels/Shell/AppViewModel.WScript.cs

```diff
@@ -94,6 +107,12 @@ private async Task RunFileValidationOnProject()
 
         foreach (var file in filesToValidate)
         {
+            if (file.Extension == ".txt")
```

Comment:
Does this check all filed in the archive folder? If yes, then the validation check should be "if extension is not a REDextension".

Reply:
Good point. What's a quick way to check that?

Reply:
We have an enum for it: ERedExtension iirc